### PR TITLE
Trim all trailing whitespace.

### PR DESCRIPTION
**Pull request by JamesMBallard**
Also add trailing newlines if there wasn't one.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,4 +12,4 @@ In this file, all the changes are listed by release.
 
 ## 100.10
 
-* Introduction of new toolkit architecture.
\ No newline at end of file
+* Introduction of new toolkit architecture.
```
Comment (JamesMBallard) [inline CHANGELOG.md:15]: I cannot tell what actually changed on this line.

**Comment by JamesMBallard** [comment]
@anmacdonald please review when you have time, merge as well if it looks ok. This is purely cosmetic and purging all trailing whitespace. I spot checked all the modifications and avoided messing with image files.

**Review by anmacdonald — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (anmacdonald) [reply CHANGELOG.md:15]: Probably a missing newline?

**Pull request merged**